src/Xamarin.Android.Build.Tasks/Xamarin.Android.Legacy.targets

```
@@ -258,14 +258,18 @@ projects. .NET 5 projects will not import this file.
           Include="@(ReferenceCopyLocalPaths)"
           Condition="'%(ReferenceCopyLocalPaths.ResolvedFrom)' != 'ImplicitlyExpandDesignTimeFacades' And '%(ReferenceCopyLocalPaths.Extension)' == '.dll' And '%(ReferenceCopyLocalPaths.RelativeDir)' == '' And Exists('%(ReferenceCopyLocalPaths.Identity)') "
       />
+      <FilteredAssemblies
+          Include="@(ReferenceDependencyPaths)"
+          Condition="'%(ReferenceDependencyPaths.ResolvedFrom)' != 'ImplicitlyExpandDesignTimeFacades' And Exists('%(ReferenceDependencyPaths.Identity)') "
+      />
       <FilteredAssemblies
           Include="@(_ReferencePath)"
           Condition="'%(ReferencePath.ResolvedFrom)' != 'ImplicitlyExpandDesignTimeFacades' And Exists('%(_ReferencePath.Identity)') "
```

This is somewhat unrelated to this PR, but I see a small mistake here, should we just fix this since I noticed it?
```suggestion
          Condition="'%(_ReferencePath.ResolvedFrom)' != 'ImplicitlyExpandDesignTimeFacades' And Exists('%(_ReferencePath.Identity)') "
```